diff --git a/node_modules/antwar/dist/build/build_worker.js b/node_modules/antwar/dist/build/build_worker.js
index 6c78843..15ede03 100644
--- a/node_modules/antwar/dist/build/build_worker.js
+++ b/node_modules/antwar/dist/build/build_worker.js
@@ -51,7 +51,7 @@ function processPage({
     const $ = cheerio.load(html);
     const components = $(".interactive").map((i, el) => {
       const $el = $(el);
-      const id = $el.attr("id");
+      const id = $el.attr("id").split('+')[0];
       const props = $el.data("props");
 
       return {
diff --git a/node_modules/antwar/src/build/build_worker.js b/node_modules/antwar/src/build/build_worker.js
index 7f2ca30..fbf685b 100644
--- a/node_modules/antwar/src/build/build_worker.js
+++ b/node_modules/antwar/src/build/build_worker.js
@@ -58,7 +58,7 @@ function processPage(
     const components = $(".interactive")
       .map((i, el) => {
         const $el = $(el);
-        const id = $el.attr("id");
+        const id = $el.attr("id").split('+')[0];
         const props = $el.data("props");
 
         return {
